src/encoded/tests/test_batch_download.py

```diff
@@ -27,3 +63,30 @@ def test_report_download(testapp, workbook):
     #     b'', b'', b'', b'', b'', b'', b''
     # ]
     assert len(lines) == 47
+
+
+def test_batch_download_files_txt(testapp, workbook):
```

Comment:
We can do a little more checking here without too much effort.  All results will have more than zero lines since the metadata_link is appended to everything.   

-assert that the first link is the metadata link.  We know that string exactly.
-assert there are more urls than just metadata link
-assert that the file urls are correct in someway...

This last one, you could do a regex to make sure all the urls fit our pattern, host_url files ENCFF... @@download ENCFF.ext.   Kinda overkill but certainly possible.  We should at least verify ONE file string format, like it starts with the correct host_url, and it contains 'files' and '@@download'.

Reply:
Thanks for this. I have addressed these validations in the code